toonz/sources/toonz/preferencespopup.cpp

```diff
@@ -1538,6 +1554,16 @@ PreferencesPopup::PreferencesPopup()
       new QLabel(tr("* Changes will take effect the next time you run Toonz"));
   note_version->setStyleSheet("font-size: 10px; font: italic;");
 
+  //--- Tablet Settings ------------------------------
+  if (showTabletSettings) categoryList->addItem(tr("Tablet Settings"));
+
+  m_enableWinInk =
+      new DVGui::CheckBox(tr("Enable Windows Ink Support* (EXPERIMENTAL)"));
```

Comment:
For now you construct the checkbox `m_enableWinInk` regardless of the `showTabletSettings` value.
If the `showTabletSettings` is false, the checkbox will be created without parent widget and thus will not be deleted along with a destruction of the `PreferencesPopup`. 

How about constructing `m_enableWinInk` only if the `showTabletSettings` is true?